### DISCUSSION TITLE
Noise Suppression

### DISCUSSION THREAD

**Discussion by benjamincoughlin** [twilio-labs/paste#2425] [Q&A] [answered]
We are adding Noise Suppression to Video & Live Video with [Krisp.ai](http://krisp.ai/) and are in need of an icon. I've done a little background research on how some headphones companies are creating this, but would anyone have some guidance to set design direction?

This is [Bose](https://assets.bose.com/content/dam/Bose_DAM/Web/consumer_electronics/global/products/headphones/noise_cancelling_headphones_700/Refresh/CTP-27267_NC700_Icon_Noise_Cancel.png/jcr:content/renditions/cq5dam.web.320.320.png):

This is[ Beats/Apple](https://www.beatsbydre.com/headphones/studio3-wireless)
<img width="552" alt="Screen Shot 2022-05-23 at 11 36 38 AM" src="https://user-images.githubusercontent.com/102981799/169885048-0aef40b8-9146-400a-8cc4-f09c23aa7634.png">

I've started exploring to have a place holder for upcoming user testing here:
https://www.figma.com/file/4GLGmNUpJrf6PYDc6Qyho6/?node-id=44%3A17334

**Comment by nkrantz** [comment]
Hey @benjamincoughlin - thanks for starting a new discussion for this. I've shared your request with the DSYS design team and we'll update this thread with more info soon!

**Comment by nkrantz** [comment]
@benjamincoughlin I chatted with the design team and they recommend [following these steps](https://paste.twilio.design/introduction/contributing/icons/#finding-an-icon). You could start by looking through [our existing icons](https://paste.twilio.design/components/icons) to see if there are any similar ones that we could tweak to add a new variant (maybe the Agent icon?). If not, you can look through Streamline to find an Icon that best suits your needs.

Are you hoping to add the Noise Suppression icon to Paste, or just use it for your product?

**Reply by benjamincoughlin** [reply]
@nkrantz This hopefully can be part of the Paste as it will be initially used in Video & Live, but if successful moved over into the Voice products.

I have built a stand for prototyping & usability testing. You can view it here:
https://www.figma.com/file/4GLGmNUpJrf6PYDc6Qyho6/?node-id=44%3A17334


![Noise Suppression   Noise Cancelation](https://user-images.githubusercontent.com/102981799/170083255-516c6a8d-bf77-46e2-96e3-9846ce95ab51.png)

![Programmable Video - Noise Suppression Example](https://user-images.githubusercontent.com/102981799/170083780-50f47123-9dfb-4fb9-b5f6-fe39d04c97ec.png)

**Comment by loreina** [comment, marked as answer]
👋 Hey @benjamincoughlin, just following up here on how to get this icon added to Paste:

1. We suggest checking [Streamline](https://www.streamlinehq.com/icons/legacy-streamline-regular) first to see if there's a similar icon we could tweak or draw inspiration from.
2. The icon will need to work at and be distinguishable at every icon size, including our smallest icon size (16x16px). We have [a list of icon size tokens on our docs site](https://paste.twilio.design/tokens/list#sizings). The external research you shared seems to use the icon on brand web pages, whereas with a product icon, it might be nice to lean towards less detail and focus on what the icon is meant to convey.
3. We generally prefer to add icons that can be more generalized into Paste. It's also totally cool to add something more product-specific, though maybe you can share more use cases for this icon or feel free to set up a time for us to chat about if this icon should live in Paste.

**Reply by andioneto** [reply]
Hi @benjamincoughlin 👋 

Checking in to see if you need more assistance here. Let us know if have questions about @loreina's suggestions above, or about something else.

Thanks!

**Reply by benjamincoughlin** [reply]
Thanks for the support and suggestions. I will ideate with some of the Streamline icons let you all know where we end up.

**Reply by kate-porter** [reply]
Sounds good, let us know how we can help!

We were also noodling on this on our side a bit, and since "noise suppression" is a somewhat complex idea to communicate via iconography alone, we played around with [a few rough ideas](https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=1508%3A340941) that don't rely on an icon. You'll also see that a couple of options group the Noise Suppression feature with the other audio controls or the "more" controls. Putting Noise Suppression right beside the meeting name gives the feature a lot of prominence in the UI, and that very well may be the intent, but if it's not, grouping it with other like-controls might be an interesting option.

Just wanted to share! Would love to hear where you land.

**Reply by serifluous** [reply]
Hey @benjamincoughlin ! Checking in to see if you were able to use any of the options @katieporter shared in your explorations.

**Reply by benjamincoughlin** [reply]
Hi @serifluous & @katieporter I really appreciate the help & suggestions. Right now the icon explorations for the Video and Live products have been deprioritized to get the APIs completed. I will let you know when it makes it back on the list of to dos, hopefully later in the year.

**Reply by loreina** [reply]
Hi @benjamincoughlin, thanks for letting us know. I'll mark the discussion as answered for now, and feel free to re-open this discussion when work on the icon explorations pick back up. Thanks!
